poupapig-backend-nestjs/.gitignore

```diff
@@ -0,0 +1,56 @@
+# compiled output
+/dist
```

Comment:
same thing here, copy this content and use it in the former .gitignore, then delete this file in this PR